Number_types/include/CGAL/GMP/Gmpq_type.h

```diff
@@ -107,6 +107,13 @@ class Gmpq
   Gmpq(unsigned long n)
   { mpq_set_ui(mpq(), n, 1); }
 
+#ifndef CGAL_CFG_NO_CPP0X_DELETED_AND_DEFAULT_FUNCTIONS
+  Gmpq(Gmpq&&) = default;
+  Gmpq(const Gmpq&) = default;
+  Gmpq& operator=(Gmpq&&) = default;
+  Gmpq& operator=(const Gmpq&) = default;
+#endif
+
```

Comment:
Hi @barcode. I wonder why you added those lines. I have tried to compile a test using `CGAL::Gmpq`, with gcc 9, `-std=c++2a`, and `-Wall -Wextra`, and I got no warning.

Reply:
It removed some warnings for me. 
Since it was an older version of gcc9 (the current gcc-master at that point in time) it might have changed now.

Reply:
I have removed this commit, and rebased your branch on `master`. Now it has only the commit that adds `export(PACKAGE CGAL)`: https://github.com/CGAL/cgal/pull/3436/commits/f924d51c6f85fd74ff1b97a477e962255e6221bc.